community-operators/cluster-logging/cluster-logging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -62,14 +62,21 @@ spec:
   displayName: Cluster Logging
 
   description: |
-    The Cluster Logging Operator for OKD provides a means for configuring and managing your aggregated logging stack.
+    # Cluster Logging
+    The Cluster Logging Operator orchestrates and manages the aggregated logging stack as a cluster-wide service.
 
-    It is recommended to deploy this operator to the **openshift-logging** namespace.
-
-    Once installed, the Cluster Logging Operator provides the following features:
+    ##Features
     * **Create/Destroy**: Launch and create an aggregated logging stack to support the entire OKD cluster.
     * **Simplified Configuration**: Configure your aggregated logging cluster's structure like components and end points easily.
 
+    ## Prerequisites and Requirements
+    ### Cluster Logging Namespace
+    Cluster logging and the Cluster Logging Operator is only deployable to the **openshift-logging** namespace. This namespace
+    must be explicitly created by a cluster administrator (e.g. `oc create ns openshift-logging`)
+    ### Elasticsearch Operator
+    The Elasticsearch Operator is responsible for orchestrating and managing cluster logging's Elasticsearch cluster.  This
+    operator must be deployed to the global operator group namespace (e.g. **openshift-operators**)
```

Comment:
@jcantrill I was refering to this section. Should this match the message in EO?

Reply:
I believe I addressed here